client/src/components/ResultPanel/ResultPanel.ts

```diff
@@ -4,49 +4,38 @@ import { Uri, ViewColumn, WebviewPanel, l10n, window } from "vscode";
 
 import { v4 } from "uuid";
 
+import { getContextValue, setContextValue } from "../ExtensionContext";
 import { isSideResultEnabled, isSinglePanelEnabled } from "../utils/settings";
 
+const SCRIPT_REGEX = /<script\b[^<]*(?:(?!<\/script>)<[^<]*)*<\/script>/gi;
+export const SAS_RESULT_PANEL = "SASResultPanel";
+
 let resultPanel: WebviewPanel | undefined;
 export const resultPanels: Record<string, WebviewPanel> = {};
```

Comment:
There's a problem. Steps to reproduce:

1. Set `"SAS.results.singlePanel": true` in settings
2. Run code to generate a result
3. Run another code to generate another result. It replaces the previous result content.
4. Right click the result content and select "Download" from the context menu.
5. Nothing happens, which is unexpected.

The reason is that the new `resultPanelId` is not recorded in `resultPanels`. Actually, I think now this `resultPanels` can be removed. The html can be always retrieved from context.